.github/workflows/deploy.yml

```diff
@@ -10,7 +10,8 @@ jobs:
     environment: Dev
     steps:
       - uses: actions/checkout@v2
```

Comment:
After this step you'll want to install node, the environment doesn't have it by default:
```
      - name: Setup node.js
        uses: actions/setup-node@v1
        with:
          node-version: '12.x'
```

This assumes you're using node version 12-ish. Update that if you're not